bindings/pulp/bindings/server.py

```diff
@@ -229,6 +235,21 @@ def _build_url(self, path, queries=()):
             path = '?'.join((path, queries))
         return path
 
+    def has_kerberos_ticket(self):
```

Comment:
This method does not seem to use self. Please add the ```@staticmethod``` decorator to it, or consider moving it to be a top level function if it is useful elsewhere. Since another module does seem to need to do this as well, it may serve well to have it be a top level function.